internal/lookout/repository/store.go

```diff
@@ -38,7 +38,7 @@ func NewSQLJobStore(db *goqu.Database, annotationPrefix string) *SQLJobStore {
 }
 
 func (r *SQLJobStore) RecordJob(job *api.Job, timestamp time.Time) error {
-	jobJson, err := json.Marshal(job)
+	jobJson, err := job.Marshal()
```

Comment:
"jobJson" isn't correct here, it is protobuf marshalling -- i'll rename

Reply:
ultimately, this was just for the out-of-date tests -- we are storing job in lookout-ingester now